inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -407,9 +451,85 @@ ngraph::op::v5::Loop::SpecialBodyPorts XmlDeserializer::parsePurposeAttribute(
     return result;
 }
 
+std::vector<std::shared_ptr<ngraph::op::util::MultiSubGraphOp::InputDescription>>
+    XmlDeserializer::parseMSInputDescription(const pugi::xml_node& node, std::string port_map_name) {
+    std::vector<std::shared_ptr<ngraph::op::util::MultiSubGraphOp::InputDescription>> inputs;
+    std::string body_name = "";
+    if (port_map_name == "then_port_map") {
+        body_name = "then_body";
+    } else if (port_map_name == "else_port_map") {
+        body_name = "else_body";
+    } else {
+        body_name = "body";
```

Comment:
why is the third case here? I would rather to raise an assert if it is used only for `If` now